kfdef/kfctl_gcp_iap.yaml

```diff
@@ -49,6 +49,13 @@ spec:
         name: manifests
         path: application/application
     name: application
+  - kustomizeConfig:
```

Comment:
Since we don't use them yet, should we wait and edit all configs later?

Reply:
Sounds good. I'll make this change.

Reply:
Sounds good. I'll make this change.